apps/root/src/services/positionService.ts

```diff
@@ -1579,33 +1550,6 @@ export default class PositionService extends EventsManager<PositionServiceData>
       }
     }
     switch (transaction.type) {
-      case TransactionTypes.newPosition: {
-        const newPositionTypeData = transaction.typeData;
-        const newId = newPositionTypeData.id;
-        if (!currentPositions[`${transaction.chainId}-${newId}-v${newPositionTypeData.version}`]) {
-          const newPosition = getNewPositionFromTxTypeData({
-            chainId: transaction.chainId,
-            id: `${transaction.chainId}-${newId}-v${newPositionTypeData.version}`,
-            positionId: BigInt(newId),
-            newPositionTypeData,
-            user: transaction.from as Address,
-          });
-
-          currentPositions[`${transaction.chainId}-${newId}-v${newPositionTypeData.version}`] = {
-            ...(currentPositions[`pending-transaction-${transaction.hash}-v${newPositionTypeData.version}`] || {}),
-            ...newPosition,
-          };
-        }
-
-        delete currentPositions[`pending-transaction-${transaction.hash}-v${newPositionTypeData.version}`];
-        this.pairService.addNewPair(
```

Comment:
We should only remove this addNewPair call, the `case TransactionTypes.newPosition` adds the new position to the UI

Reply:
Nice catch. I didn't mean to remove this. Jesus. TY 🔨